pkg/operation/seed/seed.go

```diff
@@ -601,10 +604,34 @@ func RunReconcileSeedFlow(
 
 		// Need stable order before passing the dashboards to Grafana config to avoid unnecessary changes
 		kutil.ByName().Sort(existingConfigMaps)
-
+		modifyFilter := `
+    Name          modify
+    Match         kubernetes.*
+    Condition     Key_value_matches tag __PLACE_HOLDER__
+    Add           __gardener_multitenant_id__ operator;user
+`
 		// Read all filters and parsers coming from the extension provider configurations
 		for _, cm := range existingConfigMaps.Items {
-			filters.WriteString(fmt.Sprintln(cm.Data[v1beta1constants.FluentBitConfigMapKubernetesFilter]))
+			// Remove the extensions rewrite_tag filters.
+			// TODO (vlvasilev): When all custom rewrite_tag filters are removed from the extensions this code snipped must be removed
```

Comment:
@vlvasilev are you opening PRs for the  various repos?

Reply:
ok, I will start opening PRs.

Reply:
Great, thanks a lot!